pcf/test/particle/aws/testdata.json

```diff
@@ -1,4 +1,23 @@
 {
+  "ecr-repository-apply": [
+    {
+        "pcf_name": "pcf_ecr_repository",
+        "flavor": "ecr_repository",
+        "aws_resource": {
+            "repositoryName": "gg-ecr",
+            "tags": [
+                {
+                    "Key": "lol",
+                    "Value": "cat"
+                }
+            ]
+        }
+    },
+    {},
```

Comment:
you test update tags here

Reply:
Updated testdata.json to add update ECR tags. Thanks!